packages/contracts-bedrock/snapshots/abi/OutputBisectionGame.json

```diff
@@ -198,6 +203,19 @@
     "stateMutability": "view",
     "type": "function"
   },
+  {
+    "inputs": [],
+    "name": "GENESIS_OUTPUT_ROOT",
```

Comment:
We are leaking the fact that its an immutable into the abi

Reply:
This is tracked in https://github.com/ethereum-optimism/client-pod/issues/258, and will be addressed lump-sum in another PR. There are also several other immutable vars in both the `OutputBisectionGame` + the `FaultDisputeGame` that don't follow the style guide standard.